drivers/retained_mem/Kconfig.nrf

```diff
@@ -8,3 +8,10 @@ config RETAINED_MEM_NRF_GPREGRET
 	help
 	  Enable driver for Nordic nRF GPREGRET-based retained memory
 	  register support.
+
+config RETAINED_MEM_NRF_RAM_CTRL
+	bool "nRF retention driver based on RAM control helper"
```

Comment:
imo this is to detailed description. Maybe just `nRF RAM retention driver`? User does not know and does not care what is `RAM control helper`.

Reply:
fixed